src/qibo/tests/test_callbacks.py

```diff
@@ -5,14 +5,16 @@
 import numpy as np
 from qibo.models import Circuit, AdiabaticEvolution
 from qibo import gates, callbacks
+from qibo.config import EIGVAL_CUTOFF
+
 
 # Absolute testing tolerance for the cases of zero entanglement entropy
 _atol = 1e-8
 
 
 def test_entropy_product_state():
     """Check that the |++> state has zero entropy."""
-    entropy = callbacks.EntanglementEntropy()
+    entropy = callbacks.EntanglementEntropy(compute_spectrum=True)

```

Comment:
I believe this is not need unless we also test for `entropy.spectrum` in this test.